### PR TITLE
[Documentation][CookBook] Specific SyliusBundles extension

### PR DESCRIPTION
**Pull request by Adraesh**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/cookbook/map.rst.inc [hunk 1]
```diff
@@ -22,3 +22,4 @@
 * :doc:`/cookbook/cron-jobs`
 * :doc:`/cookbook/installation-commands`
 * :doc:`/cookbook/platform-sh`
+* :doc:`/cookbook/extending-bundles-other-em`
```
Comment (lchrusciel) [inline docs/cookbook/map.rst.inc]: Missing new line at the end of file

**Comment by Adraesh** [comment]
Hi lchrusciel!

I believe it is ok now?

Let me know.

**Comment by Zales0123** [comment]
Hi Thomas, unfortunately we need another rebase here. Also you can squash some commits, as there are plenty of them and also some nasty `Merge branch` commit is here ;)

**Comment by Adraesh** [comment]
Hi @Zales0123,

Please forgive me for my lack of knowledge but into my branch I tried:

git rebase master
then
git status

and I don't have any changes to push.

Can you lead me on expected tasks to be done with GIT?

Thank you in advance.

**Comment by Zales0123** [comment]
In the most basic way you need to:
1. Fetch changes from current *master* (`git co master` and `git pull upstream master`) - `upstream` is your name of `Sylius/Sylius` remote
2. Checkout to this branch `git co docs/extending_bundle_other_em`
3. Rebase `git rebase master`
4. Solve conflicts

To have more detailed information, please, read this comment https://github.com/Sylius/Sylius/pull/8519#issuecomment-328266251

**Comment by Zales0123** [comment]
@CoderMaggie it looks useful, I suppose we still would like to merge it? If yes, will you @Adraesh be able to rebase it to current code base? Thank you :)

**Comment by Zales0123** [comment]
Closing in favor of https://github.com/Sylius/Sylius/pull/9264, one more time thank you!

**Pull request closed without merging**